lib/DOMWorld.js

```diff
@@ -203,11 +203,11 @@ class DOMWorld {
     } = options;
     // We rely upon the fact that document.open() will reset frame lifecycle with "init"
     // lifecycle event. @see https://crrev.com/608658
-    await this.evaluate(base64html => {
+    await this.evaluate(html => {
       document.open();
-      document.write(decodeURIComponent(atob(base64html).split('').map(c => '%' + ('00' + c.charCodeAt(0).toString(16)).slice(-2)).join('')));
+      document.write(html);
       document.close();
-    }, Buffer.from(html).toString('base64'));
```

Comment:
Why does this change? Or rather why was it there to begin with?

Reply:
Turns out this is another problem in protocol's encoder.

Patch is out for review: https://chromium-review.googlesource.com/c/deps/inspector_protocol/+/1621848

Once it lands and protocol rolls in both V8 and Chromium, we should re-roll.

Closing this PR until that.